slither.config.json

```
@@ -0,0 +1,4 @@
+{
+  "detectors_to_exclude": "naming-convention,solc-version",
+  "filter_paths": "(./lib|./test)"
```

Are the parentheses needed here? Since before when passed directly to the CLI it was `"./lib|./test"` without the parentheses

---

From my local testing it seems they are not needed
```suggestion
  "filter_paths": "./lib|./test"
```